build.gradle

```diff
@@ -113,7 +113,7 @@ dependencies {
     implementation group: 'org.json', name: 'json', version: '20190722'
 
 	//https://mvnrepository.com/artifact/com.nimbusds/nimbus-jose-jwt
-    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.3'
+    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.12'
```

Comment:
@SampathKumarAmex this should not break our existing vulnerable levels. Can you please check vulnerable level using this library and see if that is working as expected

Reply:
Could you please provide more info on how to test that? Do you have any documentation covering that part? 

It's good to have some test for this if feasible.

Reply:
@SampathKumarAmex we need to start VulnerableApp, go to UI, and test that each level doesn't break.

Reply:
yeah, we are lacking unit tests and other things right now.